### DISCUSSION TITLE
add javascript framework, is it possible?

### DISCUSSION THREAD

**Discussion by ericxin1982** [DioxusLabs/dioxus#1853] [General]
Hi,

Just wondering whether it supports js framework adding, like alpinejs/htmx, hypermedia html based.

Thanks
Eric Xin

**Comment by ealmloff** [comment]
If you want to embed another framework, https://github.com/DioxusLabs/dioxus/discussions/1506 and https://github.com/DioxusLabs/dioxus/discussions/251 discuss that topic in detail. You can embed another framework, but events may not be interoperable. If you can export your other framework's component as a webframework, you can pretty easily use it inside rsx. You can also embed raw htmx/html with `dangerous_inner_html` ([docs](https://dioxuslabs.com/learn/0.4/reference/rsx#the-html-escape-hatch))
